controllers/pattern_controller.go

```diff
@@ -348,10 +348,6 @@ func (r *PatternReconciler) applyDefaults(input *api.Pattern) (error, *api.Patte
 	output.Status.AppClusterDomain = clusterIngress.Spec.Domain
 	output.Status.ClusterDomain = strings.Join(ss[1:], ".")
 
-	if output.Spec.GitOpsConfig == nil {
```

Comment:
If we agree to keep gitopsconfig around, this is still needed I suspect

Reply:
For now we will leave it as is.  Need to mark this and keep it in mind in future PRs